pkg/client/app.go

```diff
@@ -137,6 +138,9 @@ func ToAppUpdate(ctx context.Context, c Client, name string, opts *AppUpdateOpti
 	if opts.AutoUpgradeInterval != "" {
 		app.Spec.AutoUpgradeInterval = opts.AutoUpgradeInterval
 	}
+	if len(opts.Memory) != 0 {
```

Comment:
nit: It seems that always setting this would be harmless. Is that the case?

Reply:
By removing the if statement here, it makes it so that if a user does not pass anything for `-m` when updating an Acorn it will overwrite the existing setting to be nothing. So this line (and the if statements before) are allowing a user to only set the options that they want to update.

Reply:
I do think that there is an update to be made here though as there is a situation where a user wants to remove all `appInstance.Spec` memory. I'll think through something to solve that issue.

Reply:
Okay so with a recent change that @jsilverio22 did we are now able to unset all memory for a Acorn by running `acorn update -m 0 foo`. This will make it so whatever memory the `foo` acorn had set will be overwritten so this code should be fine as is.